### ISSUE TITLE
Should I fork the cpan script?

### ISSUE THREAD

**Issue by briandfoy** [andk/cpanpm#187]
I have several pull requests to the code I wrote and for which people have supplied patches. These are two years old with no response to the original message, or when I pinged each of them a year ago to check if there was anything holding these up. I've emailed Andreas to ask about helping and received a response, but the situation hasn't changed. There has been a CPAN.pm release since then, but only to fix a security issue.

I'd like to apply these patches:

* #160 
* #168 
* #169
* #170 
* #171 
* #172 

My preference is that these changes be applied in this repo and be include. I don't have the commit bits to do that.

Additionally, new issues from any submitter have received no response. Looking back at all issues for the past two years, I see no responses. There are plenty of other issues that deserve attention too.

I'm a bit stuck fixing my code that is in this repo and which is included in the standard library. It looks like this repo is basically dead.

**Comment by kstarsinic** [comment]
Hear, hear!

**Comment by sjn** [comment]
Is adoption or comaint in scope?

**Comment by FaradayLight** [comment]
Given the key role of the module in Perl setups and the number of issues in the backlog, I would support any option that would facilitate the merge of patches that will address some of these issues.

**Comment by kstarsinic** [comment]
ping

**Comment by FaradayLight** [comment]
Yes, most definitely, if you have the availability. I think many would be appreciative of any efforts you can make to address known deficiencies.

**Comment by briandfoy** [comment]
I've mostly done this already in App::Cpan::Upgraded, which has applied all the patches. I've been using it locally and haven't said anything about it publicly (I dont' think).
